docs/Blog-Post.md

```diff
@@ -0,0 +1,276 @@
+# ---
+layout: blog
+title: Levanter — Towards Scaling Reproducible Foundational Models with Jax
+authors:
+    - name: David Hall
+      url: TODO
+    - name: Percy Liang
+      url: https://cs.stanford.edu/~pliang/
+display: False
+---
+> <div class="blog-tagline">
+    <strong> We introduce our projects
+        <a href="https://github.com/stanford-mercury/levanter" target="_blank">Haliax and Levanter</a>, our code
+        and infrastructure for training reproducible, legible foundation models models using Jax. We also XXX checkpoints.
+    </strong>
+> </div>
+
+
+## Introduction
+
+The growth of artificial intelligence and machine learning has brought about the need for scalable and reproducible models.
```

Comment:
Seems like we want stronger justification for reproducible, or just punt on the more detailed positioning (legibility, reproducibility, scalability) until later.

Reply:
ok, still struggling with this part, and there are a few more todos, but RFAL